base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
 
 ## Prerequisites
 
-To make the best use of this base template, you need a working knowledge of:
+To make the best use of this guide, you should have a working knowledge of:
+
+* [Git](https://git-scm.com/): The version control system used.
+* [GitHub](https://github.com/thegooddocsproject): The project hosting platform used, along with associated project management tools.
+* [GitHub Flavored Markdown](https://github.github.com/gfm/): The wiki format used to store template documentation.
+* [Currently implemented doctypes templates](https://github.com/thegooddocsproject/templates): To help identify if you need to create a new template or edit an existing one.
+* Technical writing: To produce a well-written document that includes the relevant references.
+
+## Target audience for templates
+
+The templates target multiple audiences. They include writing tips for the following roles:
+
+
+### Template author
+
+{Template author tip: Remove this “template author” section from about-{doctype}-template.md files.}
+
+A template author creates a new template in line with base and core templates.
+
+
+### Template reviewer
+
+{Template author tip: Remove this “template reviewer” section from about-{doctype}-template.md files.}
+
+A template reviewer reviews the new template against template guidelines.
+
+
+### Template customizer
+
+A template customizer customizes a doctype template to align with a specific project or organizational guidelines. For example, specify the style guide to use with the template.
+
+
+### Docset owner
+
+A docset owner establishes an Information Architecture and Documentation Management Plan for a project. Typical tasks include:
+
+* Select a set of doctype templates for the project.
+* Customize templates if needed.
+* Select a style guide for the project.
+* Establish a docset maintenance processes.
```

Comment:
```suggestion
* Establish the docset maintenance process.
```

Reply:
@Loquacity or @flicstar or someone else, I'm happy to take your advice here, and accept changes that you propose. As an impostor tech writer who's English was only just good enough to get into Engineering, my radar signal for picking up these grammatical details is pretty weak.